baseband/vdif/tests/test_vdif.py

```diff
@@ -111,6 +112,78 @@ def test_header(self):
         assert abs(header5.time - header.time - 1.*u.s) < 1.*u.ns
         assert header5['frame_nr'] == header['frame_nr']
 
+    def test_custom_header(self):
+        # Custom header with an EDV that already exists
+        with pytest.raises(ValueError):
+            @six.add_metaclass(vdif.header._VDIFHeaderRegistry)
```

Comment:
So, none of these tests need to add the metaclass (and hence you also do not need to import `six` here). In some sense, that is the beauty of metaclasses, the user doesn't have to know about them at all.